### DISCUSSION TITLE
Possible to use in combination with redirect ?

### DISCUSSION THREAD

**Discussion by Christophvh** [usernotnull/tall-toasts#17] [Q&A] [answered]
In normal laravel workflow we can send flash data directly on the redirect action like so:
`  return redirect('/dashboard')->with('status', 'Profile updated!');`

But it doesn't seem to be working with this package. My code:
```
        return redirect()->route('admin.courses.crash-courses')->with('toasts', [
            'message' => __('course :name created!', ['name' => $course->title]),
            'title' => __('Success!'),
            'type' => 'success',
        ]);
```

But the toast message is not showing? Any ideas ?

**Comment by usernotnull** [comment, marked as answer]
Hey @Christophvh,

Yep, this package doesn't handle flash messages on next page, which is a feature on my todo list :)

I'll work on it and release an update. I'm thinking of something like:

```
toast()
    ->success('`redirect` works!')
    ->pushOnNextPage();

 return redirect()->etc...
```

**Reply by Christophvh** [reply]
ok, that seems good! might be better to make the naming consistent with Laravel. In Laravel it is called reflash();
https://laravel.com/docs/8.x/session#flash-data

**Reply by usernotnull** [reply]
toast is meant to abstract laravel sessions, so laravel naming should not be an issue.

Using the global toast(), the ide will guide you with the method names, ending with either to push or pushOnNextPage... reflashing is not something related to displaying toasts, so someone who doesn't know sessions, flash and reflash will not understand what it means.... 

That's my reasoning.

**Reply by usernotnull** [reply]
ok implemented ;)

I'll write the tests and push a new release within 24 hours.

**Reply by tcox-enrollfirst** [reply]
Awesome!  We just installed this and the "redirect with toast" is our primary use case.  Anxiously awaiting... 👍

**Reply by Christophvh** [reply]
Yeah, true about the naming! Thanks for the fast response and fast fix!

**Reply by usernotnull** [reply]
v1.1.0 released 🔥

Check the [changelog](https://github.com/usernotnull/tall-toasts/blob/main/CHANGELOG.md#110---2021-10-28).
